scss/_base_typography.scss

```diff
@@ -18,7 +18,7 @@
     line-height: map-get($line-heights, default-text);
 
     @if ($increase-font-size-on-larger-screens) {
-      font-size: map-get($base-font-sizes, base);
+      font-size: 10px; //map-get($base-font-sizes, base);
```

Comment:
Is this supposed to be hardcoded to 10px?

Reply:
ah, sorry the watch task wasnm't working and I tried to make a visible change to test

Reply:
That was definitely visible, I thought something happened to zoom in my browser :D

Reply:
fixed